pkg/cmd/runbook/run/run.go

```diff
@@ -181,7 +181,7 @@ func runbookRun(cmd *cobra.Command, f factory.Factory, flags *RunFlags) error {
 		return err
 	}
 
-	parsedVariables, err := executionscommon.ParseVariableStringArray(flags.Variables.Value)
+	parsedVariables, err := executionscommon.ParseVariableStringArray(flags.Variables.Value, false)
```

Comment:
if we just remove the `enableEscape` parameter, this will work for runbook runs too